src/components/TerraformConfig/TerraformConfig.tsx

```diff
@@ -69,24 +69,31 @@ const GenerateButton = () => {
         contentClassName={styles.modal}
       >
         {error && <Alert title={error.message} />}
-        {config && checkCommands && (
+        {config && (checkCommands || probeCommands) && (
           <>
             <Alert title="Terraform and JSON" severity="info">
               The exported config is using{' '}
               <a href="https://www.terraform.io/docs/language/syntax/json.html">Terraform JSON syntax</a>. You can place
-              this config in a file with a <strong>tf.json</strong> extension and import as a module. See Terraform
-              providor{' '}
-              <a href="https://registry.terraform.io/providers/grafana/grafana/latest/docs">docs for more details</a>
+              this config in a file with a <strong>tf.json</strong> extension and import as a module. See the{' '}
+              <TextLink href="https://registry.terraform.io/providers/grafana/grafana/latest/docs" external={true}>
+                Terraform provider docs
+              </TextLink>{' '}
+              for more details
             </Alert>
             <h5>tf.json</h5>
             <Clipboard content={JSON.stringify(config, null, 2)} className={styles.clipboard} />
-            <h5>Import existing checks into Terraform</h5>
-            <Clipboard content={checkCommands.join(' && ')} className={styles.clipboard} truncate />
-            <h5>Import custom probes into Terraform</h5>
-            <Clipboard
-              content="terraform import grafana_synthetic_monitoring_probe.{{probe_name}} {{probe_id}}:{{probe_auth_token}}"
-              className={styles.clipboard}
-            />
+            {checkCommands && (
+              <>
+                <h5>Import existing checks into Terraform</h5>
+                <Clipboard content={checkCommands.join(' && ')} className={styles.clipboard} truncate/>
+              </>
+            )}
+            {probeCommands && (
+              <>
+                <h5>Import custom probes into Terraform</h5>
+                <Clipboard content={probeCommands.join(' && ')} className={styles.clipboard} truncate />
```

Comment:
I was wondering about that as well. Personally, when I copy something I like to see first what it is I'm copying and if we truncate the text it's not clear. Besides, we're limiting the max-height of the clipboard content box (allowing to scroll), so I think it might be a good idea to remove it for both of them 👍